### DISCUSSION TITLE
Maximum JSON File size - limitations

### DISCUSSION THREAD

**Discussion by selva2429** [CMSgov/price-transparency-guide#737] [Q&A] [answered]
Is there any size restrictions on the JSON files and how many splits is valid?

**Comment by shaselton-usds** [comment, marked as answer]
There are no maximum/minimum limits for either. If you find yourself looking at a larger file, we encourage leveraging streaming file reading to avoid common large file handling errors. 

The hope with "split" files is to allow for reuse of data in other MRFs to cut down on the duplicated reported rates and provider network definitions.
